### PR TITLE
initial load of integrations that are included inside the cli and proxy

### PR DESCRIPTION
**Pull request by davidmeirlevy**

### CODE REVIEW

File: packages/common/package.json [hunk 1]
```diff
@@ -45,6 +45,11 @@
   },
   "dependencies": {
     "@configu/sdk": "workspace:*",
+    "@configu-integrations/validator": "workspace:*",
+    "@configu-integrations/comptact-json": "workspace:*",
```
Comment (RichardAkman) [inline packages/common/package.json:49]: Should be `compact` instead of `comptact`

File: packages/common/package.json [hunk 2]
```diff
@@ -45,6 +45,11 @@
   },
   "dependencies": {
```
Comment (RichardAkman) [inline packages/common/package.json:46]: We're missing `configu` store here that should also be inbuilt

File: packages/common/package.json (hunk 1 above)
Comment (davidmeirlevy) [reply packages/common/package.json]: fixed.

File: packages/common/package.json (hunk 2 above)
Comment (davidmeirlevy) [reply packages/common/package.json:46]: fixed/

**Review by RichardAkman — APPROVED** [review]
LGTM

**Pull request merged**